backend/settings.py

```diff
@@ -154,12 +153,19 @@
 # So request works over the proxy
 SECURE_PROXY_SSL_HEADER = ('HTTP_X_FORWARDED_PROTO', 'https')
 
-DB_CACHE_CONFIGS = os.getenv('DB_CACHE_CONFIGS',
-                           {'CACHE_TTL': 600, 'BACKEND': 'django_mysql.cache.MySQLCache',
-                            'LOCATION': 'canvas_app_explorer_cache',
-                            'CACHE_KEY_PREFIX': 'app_explorer',
-                            'CACHE_OPTIONS': {'COMPRESS_MIN_LENGTH': 5000, 'COMPRESS_LEVEL': 6}
-                            })
+DB_CACHE_CONFIGS = os.getenv(
+    'DB_CACHE_CONFIGS',
+    {
+        'CACHE_TTL': 600,
+        'BACKEND': 'django_redis.cache.RedisCache',
+        'LOCATION': f"redis://{os.getenv('REDIS_HOST', 'redis')}:6379/1",
```

Comment:
I believe this should be more configurable and documented. 

The port and database number probably should be just part of the string. 
```suggestion
        'LOCATION': os.getenv('REDIS_URL', 'redis://redis:6379/1'),
```